sidecars/scala/Dockerfile

```diff
@@ -24,27 +24,18 @@ RUN apk --no-cache add openjdk11-jre-headless --repository=http://dl-cdn.alpinel
 
 WORKDIR /projects
 
-ENV JAVA_HOME=/usr/lib/jvm/default-jvm/ \
-    SBT_VERSION="1.3.10" \
-    METALS_VERSION="0.9.0+10-f3f3e535-SNAPSHOT"
-
-RUN mkdir -p $HOME/.cache && ln -s $HOME/.cache /root/.cache && \
```

Comment:
let this back in and the ownership change fix will work, then this folders will be available to the extension when it needs them